microsoft-365/security/office-365-security/external-email-forwarding.md

```diff
@@ -38,7 +38,7 @@ See [Configure outbound spam filtering in EOP](https://docs.microsoft.com/micros
 
 As an admin of an organization you may have company requirements to restrict or control who is able to automatically forward emails using inbox rules, or SMTP forwarding, outside of the organization. Email forwarding can be a useful feature, but can also pose a risk through the potential disclosure of information, even by providing information to attackers that can be leveraged to attack the organization or its partners.
 
-Office 365 doesn't allow automatic external forwarding by either Inbox rules or mail box configuration, which provides a secure default policy. However, the admin can modify these settings for all, or some, users in the organization. Forwarding messages between internal users isn't affected by such a modification.
+Office 365 doesn't allow automatic external forwarding by either Inbox rules or mail box configuration, which provides a secure default policy. However, the admin can modify these settings for all, or some, users in the organization. Create [Outbound spam policies](https://docs.microsoft.com/microsoft-365/security/office-365-security/configure-the-outbound-spam-policy?view=o365-worldwide&preserve-view=true#use-the-security--compliance-center-to-create-outbound-spam-policies) and modify Automatic forwarding section to control automatic email forwarding by users to external senders. This can be later apply to the internal senders that the policy applies to. Forwarding messages between internal users isn't affected by such a modification.
```

Comment:
```suggestion
Office 365 doesn't allow automatic external forwarding by either Inbox rules or mailbox configuration, which provides a secure default policy. However, the admin can modify these settings for all, or some, users in the organization. Create [outbound spam policies](https://docs.microsoft.com/microsoft-365/security/office-365-security/configure-the-outbound-spam-policy?view=o365-worldwide&preserve-view=true#use-the-security--compliance-center-to-create-outbound-spam-policies) and modify the automatic forwarding section to control automatic email forwarding by users to external senders. This can be later applied to the internal senders that the policy applies to. Forwarding messages between internal users isn't affected by such a modification.
```